### PR TITLE
Steps to create Windows Packages such as MSI

### PR DESCRIPTION
**Pull request by raghushantha**

### CODE REVIEW

File: WindowsPackages/CreatingWindowsPackage.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## Creating Windows10/Server2016 MSI Package
```
Comment (andschwa) [inline WindowsPackages/CreatingWindowsPackage.md]: I think these instructions should be under [the releasing docs](https://github.com/PowerShell/PowerShell/blob/master/docs/maintainers/releasing.md#windows).

File: WindowsPackages/CreatingWindowsPackage.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+```
+# Sync to the fork/branch code for the package
+Import-Module .\build.psm1
+Start-PSBootstrap -Package
+Start-PSBuild -Clean -CrossGen -Runtime win10-x64 -Configuration Release
+Start-PSPackage -Type msi
```
Comment (mirichmo) [inline WindowsPackages/CreatingWindowsPackage.md]: This needs 
``` powershell
Start-PSPackage -Type zip
```
as well doesn't it?

File: WindowsPackages/CreatingWindowsPackage.md (hunk 1 above)
Comment (mirichmo) [inline WindowsPackages/CreatingWindowsPackage.md]: This should be pushed from a child branch of your fork, not directly from a branch off PowerShell/master

**Review by mirichmo — CHANGES_REQUESTED** [review]

File: WindowsPackages/CreatingWindowsPackage.md (hunk 1 above)
Comment (daxian-dbw) [inline WindowsPackages/CreatingWindowsPackage.md]: I agree. These instructions should be put in the `Windows` section in `docs/maintainers/releasing.md`.

**Pull request closed without merging**